### DISCUSSION TITLE
Feature Request: Code Navigation in Rust

### DISCUSSION THREAD

**Discussion by mbartlett21** [community/community#15367] [Other Feature Feedback, Questions, & Ideas] [answered]
Rust isn't yet supported by Code Navigation (https://docs.github.com/en/repositories/working-with-files/using-files/navigating-code-on-github).

**Comment by look** [comment, marked as answer]
Hi @mbartlett21, Rust navigation is supported in the new code search preview (https://cs.github.com/home). It's currently a Technology Preview, but we're working on bringing it into GitHub.com.
